web/src/main/java/com/futurewei/alcor/web/entity/node/NmmInfo.java

```diff
@@ -0,0 +1,21 @@
+package com.futurewei.alcor.web.entity.node;
+
+public class NmmInfo {
+    public NmmInfo(String ncmId, NcmInfo ncmInfo, NodeInfo nodeInfo) {
+        this.ncmId = ncmId;
+        this.ncmInfo = ncmInfo;
+        this.nodeInfo = nodeInfo;
+    }
```

Comment:
@pkommoju If you declare in this way, NmmInfo can only contain 1 Ncm and 1 Node mapping.

Reply:
That is a glaring mistake. What is needed is a multiset, one key multiple values. I see in other instances, we do use a list as value. So I guess, I will change to list then.

Reply:
This change made ICacheRepository unusable because it's interface doesn't allow appending items of type other than <T>. I will have to switch to ICache member and expose the right API.

Reply:
NmmInfo now contains list of node ids. changes pushed.